cmd/edgectl/aes_install_errors.go

```diff
@@ -256,29 +297,33 @@ func (i *Installer) AESACMEChallengeError(err error) Result {
 // Unable to make an HTTP Post to Metriton at https://metriton.datawire.io/register-domain
 // and so cannot acquire a DNS name for the cluster's load balancer.
 func (i *Installer) DNSNamePostError(err error) Result {
+	url := "https://www.getambassador.io/docs/latest/topics/install/help/dns-name-post"
 	i.Report("dns_name_failure", ScoutMeta{"err", err.Error()})
 
 	return Result{
-		URL: "https://www.getambassador.io/docs/latest/topics/install/help/dns-name-post",
-		Err: errors.Wrap(err, "acquire DNS name (post)"),
+		ShortMessage: "Failed to register DNS name for the current installation",
+		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about acquiring a DNS name to continue installing Ambassador Edge Stack.", url),
+		URL:          url,
+		Err:          errors.Wrap(err, "Failed to acquire DNS name (post)"),
 	}
 }
 
 // Unable to fetch the response from the HTTP Post to Metriton.
 func (i *Installer) DNSNameBodyError(err error) Result {
+	url := "https://www.getambassador.io/docs/latest/topics/install/help/dns-name-body"
 	i.Report("dns_name_failure", ScoutMeta{"err", err.Error()})
 
-	url := "https://www.getambassador.io/docs/topics/install/help/dns-name-body"
 	return Result{
 		ShortMessage: "Failed to register DNS name for the current installation",
 		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about acquiring a DNS name to continue installing Ambassador Edge Stack.", url),
 		URL:          url,
-		Err:          errors.Wrap(err, "acquire DNS name (read body)"),
+		Err:          errors.Wrap(err, "Failed to acquire DNS name (read body)"),
```

Comment:
It's unclear to me why some of these `errors.Wrap(...)` calls include a brief error phase and others do not... is that what's contributing to the redundancy Bjorn saw?

Reply:
probably, I will take a look.  I understood that we weren't going to worry about the redundancy this time around but if this is something that we want to address for 1.4.1 we can do that.

Reply:
We don't need to address it for 1.4.1, was just thinking out loud here.